diff --git a/source/atelier/script/world/world.d b/source/atelier/script/world/world.d
index 26846bc..df97b89 100644
--- a/source/atelier/script/world/world.d
+++ b/source/atelier/script/world/world.d
@@ -8,6 +8,7 @@ import atelier.render;
 import atelier.world;
 import atelier.ui;
 import atelier.script.util;
+import atelier.audio;
 
 package void loadLibWorld_world(GrModule mod) {
     mod.setModule("world.world");
@@ -108,6 +109,20 @@ package void loadLibWorld_world(GrModule mod) {
     mod.addStatic(&_findByTag, worldType, "findEntitiesByTag",
         [grString], [grList(entityType)]);
 
+
+    mod.setDescription(GrLocale.fr_FR,
+        "Lance un lecteur audio");
+    mod.setParameters(["audio resource"]);
+    mod.addStatic(&playSound, worldType, "playSound", [
+            grString
+        ]);
+
+}
+
+
+private void playSound(GrCall call) {
+    Sound sound = Atelier.res.get!Sound(call.getString(0));
+    Atelier.audio.play(new SoundPlayer(sound));
 }
 
 private void _addEntity(GrCall call) {
